CHANGELOG.md

```diff
@@ -4,7 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [1.15.0] - 2021-11-08
+### Added
+- Add `fx.Annotate` to allow users to provide parameter and result tags easily without
+  having to create an `fx.In` or `fx.Out` structs.
+- Add `fx.As` to allow users to provide a constructor as an interface that its result
```

Comment:
...that allows users to provide a constructor for a type via an interface that the type implements, as opposed to the type itself.